inquire.js

```diff
@@ -57,6 +69,9 @@ const query = (reader, question, answers) => {
                 reject();
             }
         });
+        if (previousAnswer) {
```

Comment:
Is the intention of this to make it so that the previous answer is auto-populated? i.e.,

```shell
Do you want bacon (Y/n)? Yes
```